recipes/ccache/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
It doesn't really matter in cmake wrapper of the recipe, why do you want TARGETS?